src/main/java/com/nhantran/pages/ForgetPasswordPage.java

```diff
@@ -4,12 +4,12 @@
 import org.openqa.selenium.By;
 
 public class ForgetPasswordPage extends BasePage {
-    By txtEmail = By.xpath("//input[@id='email']");
-    By sendInstructionButton = By.xpath("//input[@type='submit']");
+    private By txtEmail = By.xpath("//input[@id='email']");
```

Comment:
Should be consistent when defining variables, base their names on the type of element (textbox, combobox, button, label, combobox,...):
1. Use abbreviated names: txtEmail, btnSendInstructions, lblSuccessMessage, etc.
2. Use full names indicating the type of element: emailTextBox, sendInstructionsButton, etc.
Ensure consistency across pages